daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -345,6 +350,80 @@ allocateParty args name = do
         setRequestBodyJSON $ AllocatePartyRequest {identifierHint = text, displayName = text}
   return party
 
+tokenFor :: [L.Party] -> L.LedgerId -> L.Token
```

Comment:
Don't we have a function like this already somewhere else?

Reply:
Yes, we have one in the integration tests. I now reuse this one.